bot.py

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+"""Code for the telegram-bot track."""
+
+import datetime
+import logging
+import os
+
+import ephem
+from telegram.bot import Bot
+from telegram.ext import Updater, CommandHandler, MessageHandler, Filters
+from telegram.update import Update
+
+logging.basicConfig(format='%(name)s - %(levelname)s - %(message)s',
+                    level=logging.INFO)
+
+
+def main():
+    """Run the bot."""
+    token = os.environ['TG_TOKEN']
+    proxy = {'proxy_url': os.environ['PROXY_URL'],
+             'urllib3_proxy_kwargs': {
+                 'username': os.environ['PROXY_USER'],
+                 'password': os.environ['PROXY_PASS'],
+             },
+             }
```

Comment:
hmmm formatting here is kinda weird. Both braces are on the same indentation level though they're not on the same logical level. I would change the indentation therefore.

Try `black` formatter just out of curiosity? See if fixes this thing

Reply:
Fixed this with a setting in PyCharm code formatter.